src/App.js

```diff
@@ -751,10 +751,6 @@ class MainMap extends React.Component {
             this.boardTable = new Uint32Array(buf);
             // Warning: Do I need to await wasm here first?
             console.log('Board table length:', this.boardTable.length);
-            // Make sure every value is in range.
```

Comment:
reasonable to remove this; maybe in the future do a hash check?

Reply:
Yeah, that could make sense. We probably won't want to block on it as we did with this one, just since corruption would be so rare.